wp-content/themes/aerospace/tag-space-threats.php

```diff
@@ -0,0 +1,96 @@
+<?php
+/**
+ * The template for displaying archive pages
+ *
+ * @link https://codex.wordpress.org/Template_Hierarchy
+ *
+ * @package Aerospace
+ */
+
+$archive_query = get_posts($wp_query->query_vars);
+
+if ( get_archive_top_content() ) {
+    $description = get_archive_top_content();
+} else {
+    $description = get_the_archive_description();
```

Comment:
On this particular archive page, we only need to use `$description = get_the_archive_description()`. The other part of the if statement isn't relevant here, and we don't use `$description_extra`, so that can also be removed.